### ISSUE TITLE
no-superfluous-tags does not work with example tags

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#167]
In the same way that

``` gherkin
@tag1
Feature: Feature with superfluous tags

@tag1
Scenario: This is a Scenario name
  Then this is a then step

Scenario: This is another Scenario name
  Then this is a then step
```
is reporting an error because of `tag1` exists at feature and scenario level, the following gherkin code

``` gherkin
Feature: Feature with superfluous tags

Scenario: This is a Scenario name
  Then this is a then step

@tag1
Scenario Outline: This is a Scenario name
  Then this is a then step with variable <foo>
Examples:
  | foo |
  | bar |
@tag1
Examples:
  | foo |
  | rab |
```

should report an error because `tag1` exists at scenario and example level.
